pkgs/tools/networking/debian-hostname/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "debian-hostname";
+  version = "3.23";
+
+  src = fetchurl {
+    url = "http://deb.debian.org/debian/pool/main/h/hostname/hostname_${version}.tar.gz";
+    sha256 = "sha256-vG0ZVLIoSYaf+LKmAuOfCLFwL2htS1jdeSfN61tIdu8=";
+  };
+
+  outputs = ["out"];
+
+  patches = [];
+
+  nativeBuildInputs = [];
+  buildInputs = [];
+  doCheck = false;
+  installFlags = [];
+  installPhase = ''
+    sed -i.orig '/install/ s,-[og] root,,g' Makefile
+    make install BINDIR=$out/bin MANDIR=$out/share/man
+  '';
+
+  meta = with lib; {
+    description = "Collection of common network programs";
+
+    longDescription = ''
+      utility to set/show the host name or domain name
+    '';
+
+    homepage = "https://tracker.debian.org/pkg/hostname";
+    license = licenses.gpl2Plus;
+
+    maintainers = with maintainers; [ ];
+    platforms = platforms.gnu;
```

```suggestion
    description = "Collection of network utilities to set/show the host name or domain name";
    homepage = "https://tracker.debian.org/pkg/hostname";
    license = licenses.gpl2Plus;
    maintainers = with maintainers; [ ];
    platforms = platforms.gnu;
```
meta.maintainers must be set.